### DISCUSSION TITLE
Better overflow/wraping needed for long workflow names

### DISCUSSION THREAD

**Discussion by shtrom** [community/community#72693] [Actions]
### Select Topic Area

Product Feedback

### Body

Hey there,

We have a number of templated actions which vary in terms of there deployment target. Unfortunately, the name of the target is at the end of the workflow name, and, well, they are hidden in the UI because of overflow.

Here's an example:

<img width="1352" alt="Screenshot 2023-10-27 at 15 02 00" src="https://github.com/community/community/assets/160280/517003af-3e2c-4498-bc33-72442743a6bc">

Not great to distinguish which one's which. In addition, there is no `title` or other tooltip expanding the full name on hover, so it's impossible to tell which entry is what without cliking on each.

It would be better if there was some wrapping of those long entries, so they are displayed in full.

**Comment by Sophist-UK** [comment]
At a minimum can we please have tooltips containing the entire job name.

**Reply by jithunnair-amd** [reply]
I'd like to request this too. Github Actions is used as the CI/CD platform for the PyTorch project, and a lot of the workflows have jobs with long names that have the same starting string: https://github.com/pytorch/pytorch/actions/runs/9290224474/job/25565986312

Not having tooltips, or the ability to resize the left sidebar means that we have to click on each job to see if it's the one we want.

**Comment by grandmas-favorite** [comment]
Hi @shtrom!

We appreciate your feedback and your valuable input. We want to be transparent about our current priorities. You can gain insights into the projects we’re currently focusing on for the upcoming year by checking our public roadmap. While smaller fixes may not always be explicitly listed, they often get addressed when related work is in progress.

While we do review all incoming feedback unfortunately due to the volume of feedback, we may not be able to respond to each one individually at this time. However, we strongly encourage you to continue providing your valuable input as it greatly helps us prioritize and improve GitHub.

Your contributions are instrumental in making GitHub better for all users!

**Comment by rgonzalezfluendo** [comment]
Hi @grandmas-favorite,

Thanks for the information. It's a simple feature to implement with a valuable impact.

Let me share my custom implementation I use while waiting for an official solution:

```js
Array.from(document.getElementsByClassName('ActionListItem-label--truncate')).forEach((e) => e.title = e.innerHTML.trim())
```


[Screencast from 2024-06-05 22-54-14.webm](https://github.com/community/community/assets/56379722/039bf37c-c6db-4c24-953d-64a409050f41)

**Comment by lefteris-tsallas-leanix** [comment]
The [Refined Github chrome extension](https://github.com/refined-github/refined-github) can be used as a workaround until this is officially supported. It un-truncates the long job names. More details in the related issue [un-truncates the long names](https://github.com/refined-github/refined-github/issues/8372)
